### PR TITLE
fix 2 warnings in CodeAnalysis

### PR DESCRIPTION
**Pull request by heng-liu**
## Bug

Fixes: https://github.com/NuGet/Home/issues/9546
Regression: No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: fix 2 warnings raised by CodeAnalysis.FxCopAnalyzer

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

File: src/NuGet.Core/NuGet.Packaging/Signing/Cms/ManagedCmsWrapper.cs [hunk 1]
```diff
@@ -47,7 +47,10 @@ public void AddCertificates(IEnumerable<X509Certificate2> certificates)
         {
             foreach (var cert in certificates)
             {
-                _signedCms.AddCertificate(new X509Certificate2(cert));
+                using (var certificate = new X509Certificate2(cert))
+                {
+                    _signedCms.AddCertificate(certificate);
+                }
```
Comment (zivkan) [inline src/NuGet.Core/NuGet.Packaging/Signing/Cms/ManagedCmsWrapper.cs]: By putting this is a using block, it means the certificate that was added to `_signedCms` is now disposed. Are you sure that's ok?

Comment (heng-liu) [reply src/NuGet.Core/NuGet.Packaging/Signing/Cms/ManagedCmsWrapper.cs]: Thanks for reviewing! 
Looks like the `AddCertificate `method will create a copy of the certificate in following code:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/SignedCms.cs#L639-L671

Comment (zivkan) [reply src/NuGet.Core/NuGet.Packaging/Signing/Cms/ManagedCmsWrapper.cs]: ~Consider adding a comment saying that `AddCertificates` copies the cert.~

As you pointed out, there's no point making a new cert.

Comment (heng-liu) [reply src/NuGet.Core/NuGet.Packaging/Signing/Cms/ManagedCmsWrapper.cs]: So it should be ok not new any certificate, just use the one from parameter. 
I'll fix it. Thanks!

**Review by zivkan — APPROVED** [review]

**Pull request merged**